docs/specs/localization.yml

```diff
@@ -697,6 +770,10 @@ repos:
             bilingual: true
   https://github.com/ss/c.zh-cn#master-sxs:
     - files:
+        docfx.yml: |
+          # TODO: should be able to figure out bilingual automatically using convention
```

Comment:
This case illustrate that when build form loc, we currently aren't using convention to figure out if it is sxs